its/projects/SymbolicLinks/Links/LinkedSample.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace SymbolicLinks;
```

Comment:
The link vs target confuses me a little. Can we rename the directory from `Links` to `Linked` or `LinkedFiles`.